src/main/java/ContainsTrue.java

```diff
@@ -0,0 +1,16 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class contrainsTrue {
+    WebDriver driver;
+
+    public contrainsTrue(WebDriver driver){
+        this.driver=driver;
+    }
+
+    public String displayedText(){
+       return driver.findElement(By.id("productTitle")).getText();
```

Comment:
can you move the selector as a class member?

Reply:
It was done on the last commit